src/main.ts

```diff
@@ -53,15 +55,22 @@ export async function initRemix({
     app.whenReady(),
   ])
 
+  let lastBuildTime: number = 0;
+  let buildPath = typeof serverBuildOption === "string" ? join(serverBuildOption, 'index.js') : join(process.cwd(), 'desktop/build.js')
+
   protocol.interceptBufferProtocol("http", async (request, callback) => {
     try {
+      let buildTime = 0;
       if (mode === "development") {
         assetFiles = await collectAssetFiles(publicFolder)
+        let buildStat = await stat(buildPath)
+        buildTime = buildStat.mtimeMs;
```

Comment:
Adding onto the last comment, only stat this if the build path is defined. I would put this in its own condition for clarity.
```ts
let buildTime = 0
if (mode === "development" && buildPath !== undefined) {
  const buildStat = await stat(buildPath)
  buildTime = buildStat.mtimeMs
}
```